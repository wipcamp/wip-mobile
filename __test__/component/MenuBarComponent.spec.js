/**
 * @jest-environment node
 */

import React from 'react'
import renderer from 'react-test-renderer'

import MenuBar from '../../components/MenuBarComponent'

it('rendering', () => {
    const tree = renderer
        .create(
            <MenuBar>Test</MenuBar>
        )
        .toJSON()
    expect(tree).toMatchSnapshot()
})

it('rendering top', () => {
    const tree = renderer
        .create(
            <MenuBar top>Test</MenuBar>
        )
        .toJSON()
    expect(tree).toMatchSnapshot()
})