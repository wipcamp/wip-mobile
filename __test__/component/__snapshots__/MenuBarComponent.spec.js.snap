// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 0.275,
      },
      Object {
        "flexDirection": "row",
      },
      Object {
        "justifyContent": "space-between",
      },
      Object {
        "paddingTop": 10,
      },
      Object {
        "paddingBottom": 15,
      },
      Object {
        "paddingLeft": 5,
        "paddingRight": 5,
      },
    ]
  }
>
  Test
</View>
`;

exports[`rendering top 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 0.275,
      },
      Object {
        "flexDirection": "row",
      },
      Object {
        "justifyContent": "space-around",
      },
      Object {
        "paddingTop": 15,
      },
      Object {
        "paddingBottom": 10,
      },
    ]
  }
>
  Test
</View>
`;
